teuthology/test/integration/openstack-integration.py

```diff
@@ -0,0 +1,275 @@
+from cStringIO import StringIO
+import argparse
+import logging
+import json
+import os
+import tempfile
+import testtools
+import shutil
+import subprocess
+import sys
+
+import teuthology.lock
+import teuthology.nuke
+import teuthology.misc
+import teuthology.schedule
+import teuthology.suite
+import scripts.schedule
+import scripts.lock
+import scripts.suite
+from teuthology import provision
+from teuthology.config import config
+
+
+class Integration(testtools.TestCase):
```

Comment:
We're pretty much standardized on ``py.test``; is there some reason to use ``testtools`` here instead?